library/core/tests/lib.rs

```diff
@@ -1,115 +1,119 @@
+// tidy-alphabetical-start
```

Comment:
This cleanup should be in a separate PR.

Reply:
Fair, I was mostly just tired that every single change to this file was effectively a merge conflict, and thought that if I was going to have to fix merge conflicts every time, I might as well fix the file.

Reply:
I did what I do best and substantially increased the scope of this change and moved it to another PR: #114766 

Will rebase once that's merged (in some form or another), so, I'll mark this as blocked for now even though I technically don't need to block on that.

Reply:
And now I'm putting that on hold, so, I'll remove this bit and rebase.